### PR TITLE
v52 release notes

### PR DESCRIPTION
**Pull request by hikchoi**
This PR:
- Adds release notes for v52.
- Adds officie hour note to reference in release note.
- Fixes title of v51 release notes.

### CODE REVIEW

File: vault/dendron.release.2021-07-26.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- Hashtag autocomplete improvements ([[docs|dendron.topic.tags#autocomplete-intellisense]])
+- Doctor command to automatically fix the frontmatter ([[docs|dendron.topic.commands#fixfrontmatter]])
+- Google Doc Import Pod ([[docs|dendron.topic.pod.builtin.google-docs.import]])
+- Ability to control graph zoom sensitivity 
+- Bug Fixes
+  - Reduce size of dendron-cli
+  - Hashtag links duplicate following text
+  - Issues with parsing wikilinks with unicode and special symbols
+  - Autocomplete takes a long time
+- A lot of interal improvements
```
Comment (kpathakota) [inline vault/dendron.release.2021-07-26.md:25]: sp: Internal

File: vault/dendron.release.2021-07-19.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 id: 9rqhTCE1dAUQJ3fD
-title: 2021 07 19
+title: Release Notes (version 0.51)
```
Comment (kpathakota) [inline vault/dendron.release.2021-07-19.md:3]: Great catch!

**Review by kpathakota — APPROVED** [review]

File: vault/dendron.release.2021-07-26.md (hunk 1 above)
Comment (hikchoi) [reply vault/dendron.release.2021-07-26.md]: fixed!

**Review by kevinslin — APPROVED** [review]
this is great. awesome work with your first release notes :)

**Pull request merged**